api-gateway/caching-with-redis/modules/redis/src/main/java/io/knotx/example/redis/util/JsonSerializer.java

```diff
@@ -0,0 +1,37 @@
+package io.knotx.example.redis.util;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+public class JsonSerializer {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(JsonSerializer.class);
+
+  private JsonSerializer() {
+    // util
+  }
+
+  public static String serializeObject(Object object) {
+    if (object instanceof JsonObject) {

```

Comment:
you can use `.toString`
since `toString` implementations of `JsonArray` and `JsonObject` both invoke their own `.encode` methods